### DISCUSSION TITLE
Min SDK version(currently 28)

### DISCUSSION THREAD

**Discussion by JohnettJobben1** [Canardoux/flutter_sound#1132] [Q&A] [answered]
Is it possible to bring down this([Android minSdkVersion](https://github.com/Canardoux/flutter_sound/blob/02fa06c5f982de819f7de201011510888a578b96/flutter_sound/android/build.gradle#L36)) to 24?

In the comment it's mentioned like following. 

`18 works fine for the player, but the recorder needs at least 24`

Could you please share the reason why we need to upgrade to version 28, rather than using version 24, which is the minimum required by the recorder?

**Comment by Larpoux** [comment, marked as answer]
There is no real reason : I had to upgrade the version because of #1129
### 9.17.4
- Android : minSdkVersion 28 // 18 works fine for the player, but the recorder needs at least 24 (was 18) ([#1129](https://github.com/Canardoux/flutter_sound/issues/1129))
- Android : buildToolsVersion “34.0.0” (added) ([#1129](https://github.com/Canardoux/flutter_sound/issues/1129))

I don't really master the version problem we had with #1129.
I am going to try to downgrade to 24 if it is OK with @izmeera2000 and @joshua1996, this afternoon, and I will post something to you when done.
Sorry for the inconvenience.

**Reply by JohnettJobben1** [reply]
Thank you so much for the quick response and the clarification! 

I really appreciate it. If there's nothing preventing us from downgrading to version 24(https://github.com/Canardoux/flutter_sound/issues/1129 ---> If this requires minSdkVersion 28, then let's keep it that way.), it would be great to proceed with that. It would also allow us to support some lower versions, which could be beneficial. 

Thanks again for your help, and also for this wonderful library.

**Comment by Larpoux** [comment]
Flutter Sound 9.17.5 is released. I cross my fingers that it has no impact on #1129
### 9.17.5

- Try to downgrade the Android minSdkVersion to 24 ([#1132](https://github.com/Canardoux/flutter_sound/discussions/1132)) (we must check with @izmeera2000 and @joshua1996 that's it is OK with ([#1129](https://github.com/Canardoux/flutter_sound/issues/1129)) )

**Reply by JohnettJobben1** [reply]
@Larpoux Thank you for your prompt action.

@izmeera2000 and @joshua1996 If possible, could you please confirm if this works for you?
